data_managers/data_manager_hisat_index_builder/tool-data/hisat_indexes.loc.sample

```diff
@@ -0,0 +1,37 @@
+# hisat_indices.loc.sample
+# This is a *.loc.sample file distributed with Galaxy that enables tools
+# to use a directory of indexed data files. This one is for HISAT.
+# See the wiki: http://wiki.galaxyproject.org/Admin/NGS%20Local%20Setup
+# First create these data files and save them in your own data directory structure.
+# Then, create a hisat_indexes.loc file to use those indexes with tools.
+# Copy this file, save it with the same name (minus the .sample), 
+# follow the format examples, and store the result in this directory.
+# The file should include an one line entry for each index set.
+# The path points to the "basename" for the set, not a specific file.
+# It has four text columns seperated by TABS.
+#
+# <unique_build_id>	<dbkey>	<display_name>	<file_base_path>
+#
+# So, for example, if you had hg18 indexes stored in:
+#
+#    /depot/data2/galaxy/hg19/hisat/
+#
+# containing hg19 genome and hg19.*.bt2 files, such as:
```

Comment:
So it's similar to bowtie2's indexes, but trying to use bowtie2-build generated indexes was a massive failure.